src/browser/BrowserPasswordGeneratorWidget.cpp

```diff
@@ -0,0 +1,346 @@
+/*
```

Comment:
Also, all the config/preference should stay in one place (PasswordGeneratorWidget) and not being duplicated